reference-architecture/gce-cli/ansible/playbooks/roles/openshift-ansible-installer/tasks/main.yaml

```diff
@@ -0,0 +1,6 @@
+---
+- name: download openshift ansible installer from github
+  git:
+    repo: https://github.com/openshift/openshift-ansible.git
+    dest: '{{ playbook_dir }}/../../../../../openshift-ansible'
```

Comment:
It is quite ugly, but you can use the same file path and mess with symlinks... so if it is RHEL, create a symlink to /usr/share/ansible/openshift-ansible/playbooks/... otherwise, leave it as it should.